drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -341,17 +348,18 @@ end
 local function find_default_endpoint(device, component)
   local switch_eps = device:get_endpoints(clusters.OnOff.ID)
   local button_eps = device:get_endpoints(clusters.Switch.ID, {feature_bitmap=clusters.Switch.types.Feature.MOMENTARY_SWITCH})
-  local all_eps = {}
+  table.sort(switch_eps)
+  table.sort(button_eps)
```

Comment:
Why are we doing this work twice, since this same sorting is accomplished in `initialize_switch`? 

Further, the `component` field is not used here- we could remove it and replace it with an opts field that includes the presorted switch and button endpoint table.

Reply:
I removed the component field, but since `initialize_switch` is not guaranteed to run before `find_default_endpoint`, for existing device for example, I think we should leave this sorting here.